### ISSUE TITLE
issues pulling gloo fed helm chart

### ISSUE THREAD

**Issue by jenshu** [solo-io/gloo#7463]
### Gloo Edge Version

1.12.x (latest stable)

### Describe the bug

Some gloo fed helm charts cannot be pulled, e.g.
```
helm install -n gloo-system gloo-fed gloo-fed/gloo-fed --create-namespace --set-string license_key=$GLOO_LICENSE_KEY --version 1.12.36
```
results in:
```
Error: failed to download "gloo-fed/gloo-fed" at version "1.12.36" (hint: running `helm repo update` may help)
```

There seems to be a bug with how we are uploading the charts and index to the google cloud bucket.

### Steps to reproduce the bug

install gloo fed

### Expected Behavior

should be able to download chart successfully

**Comment by jenshu** [comment]
Charts should be fixed going forward, starting from GlooEE v1.13.0-beta12, v1.12.37, v1.11.50, and v1.10.34
We still need to do some work to repair the existing charts.

**Comment by jenshu** [comment]
all the old charts should now be pullable too
